atom/browser/api/lib/browser-window.coffee

```diff
@@ -40,8 +40,9 @@ BrowserWindow::_init = ->
   @once 'closed', =>
     BrowserWindow.windows.remove @id if BrowserWindow.windows.has @id
 
-BrowserWindow::openDevTools = ->
-  @_openDevTools()
+BrowserWindow::openDevTools = (options={}) ->
+  dock = if options.detach then false else true
+  @_openDevTools(dock)
```

Comment:
Ah I forgot that.

I think we can do `options.detach ?= false` before calling `@_openDevTools`, this matches the style of how we do with default options in other places.